cli/test/org/partiql/cli/functions/ReadFileTest.kt

```diff
@@ -77,7 +77,43 @@ class ReadFileTest {
         val actual = function.call(env, args).ionValue
         val expected = "[{_1:\"1\",_2:\"2\"}]"
 
-        assertEquals(actual, ion.singleValue(expected))
+        assertEquals(ion.singleValue(expected), actual)
+    }
+
+    @Test
+    fun readCsvWithDoubleQuotesEscape() {
+        writeFile("data_with_double_quotes_escape.csv", "\"1,2\",2")
+
+        val args = listOf("\"${dirPath("data_with_double_quotes_escape.csv")}\"", "{type:\"csv\"}").map { it.exprValue() }
```

Comment:
Can the user specify Ion symbols instead of strings?  (I think they should.) Need at least one test for that too.